libfastmem/Makefile

```diff
@@ -0,0 +1,21 @@
+# Port Metadata
+PORTNAME = 			libfastmem
+PORTVERSION = 		1.0.0
+
+MAINTAINER =        Nobody
```

Comment:
I suspect this shouldn't be listed as `Nobody`, but rather you, or at least whoever is responsible for the upstream repo for the code.

Reply:
You are right, as I'm the owner of the repo, let's put my name.